### ISSUE TITLE
Cannot create provider for file (Argument 3 passed to Sonata\MediaBundle\Provider\Metadata)

### ISSUE THREAD

**Issue by byhaskell** [sonata-project/SonataMediaBundle#1533]
#### Sonata packages

```
sonata-project/admin-bundle              3.45.1 = 3.45.1 The missing Symfony Admin Generator
sonata-project/block-bundle              3.14.0 = 3.14.0 Symfony SonataBlockBundle
sonata-project/cache                     2.0.1  = 2.0.1  Cache library
sonata-project/core-bundle               3.15.0 = 3.15.0 Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.4.0  = 2.4.0  Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.1.5  = 1.1.5  Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.8.0  = 3.8.0  Symfony Sonata / Integrate Doctrine ORM into the SonataAdm...
sonata-project/easy-extends-bundle       2.5.0  = 2.5.0  Symfony SonataEasyExtendsBundle
sonata-project/exporter                  2.0.0  = 2.0.0  Lightweight Exporter library
sonata-project/media-bundle              3.18.0 = 3.18.0 Symfony SonataMediaBundle
```

#### Symfony packages

```
symfony/asset                v4.2.2  = v4.2.2  Symfony Asset Component
symfony/cache                v4.2.2  = v4.2.2  Symfony Cache component with PSR-6, PSR-16, and tags
symfony/config               v4.2.2  = v4.2.2  Symfony Config Component
symfony/console              v4.2.2  = v4.2.2  Symfony Console Component
symfony/contracts            v1.0.2  = v1.0.2  A set of abstractions extracted out of the Symfony components
symfony/debug                v4.2.2  = v4.2.2  Symfony Debug Component
symfony/debug-bundle         v4.2.2  = v4.2.2  Symfony DebugBundle
symfony/dependency-injection v4.2.2  = v4.2.2  Symfony DependencyInjection Component
symfony/doctrine-bridge      v4.2.2  = v4.2.2  Symfony Doctrine Bridge
symfony/dotenv               v4.2.2  = v4.2.2  Registers environment variables from a .env file
symfony/event-dispatcher     v4.2.2  = v4.2.2  Symfony EventDispatcher Component
symfony/expression-language  v4.2.2  = v4.2.2  Symfony ExpressionLanguage Component
symfony/filesystem           v4.2.2  = v4.2.2  Symfony Filesystem Component
symfony/finder               v4.2.2  = v4.2.2  Symfony Finder Component
symfony/flex                 v1.1.8  = v1.1.8  Composer plugin for Symfony
symfony/form                 v4.2.2  = v4.2.2  Symfony Form Component
symfony/framework-bundle     v4.2.2  = v4.2.2  Symfony FrameworkBundle
symfony/http-foundation      v4.2.2  = v4.2.2  Symfony HttpFoundation Component
symfony/http-kernel          v4.2.2  = v4.2.2  Symfony HttpKernel Component
symfony/inflector            v4.2.2  = v4.2.2  Symfony Inflector Component
symfony/intl                 v4.2.2  = v4.2.2  A PHP replacement layer for the C intl extension that includes addit...
symfony/maker-bundle         v1.11.3 = v1.11.3 Symfony Maker helps you create empty commands, controllers, form cla...
symfony/options-resolver     v4.2.2  = v4.2.2  Symfony OptionsResolver Component
symfony/polyfill-intl-icu    v1.10.0 = v1.10.0 Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring    v1.10.0 = v1.10.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php72       v1.10.0 = v1.10.0 Symfony polyfill backporting some PHP 7.2+ features to lower PHP ver...
symfony/process              v4.2.2  = v4.2.2  Symfony Process Component
symfony/profiler-pack        v1.0.4  = v1.0.4  A pack for the Symfony web profiler
symfony/property-access      v4.2.2  = v4.2.2  Symfony PropertyAccess Component
symfony/routing              v4.2.2  = v4.2.2  Symfony Routing Component
symfony/security-acl         v3.0.1  = v3.0.1  Symfony Security Component - ACL (Access Control List)
symfony/security-bundle      v4.2.2  = v4.2.2  Symfony SecurityBundle
symfony/security-core        v4.2.2  = v4.2.2  Symfony Security Component - Core Library
symfony/security-csrf        v4.2.2  = v4.2.2  Symfony Security Component - CSRF Library
symfony/security-guard       v4.2.2  = v4.2.2  Symfony Security Component - Guard
symfony/security-http        v4.2.2  = v4.2.2  Symfony Security Component - HTTP Integration
symfony/stopwatch            v4.2.2  = v4.2.2  Symfony Stopwatch Component
symfony/swiftmailer-bundle   v3.2.5  = v3.2.5  Symfony SwiftmailerBundle
symfony/templating           v4.2.2  = v4.2.2  Symfony Templating Component
symfony/translation          v4.2.2  = v4.2.2  Symfony Translation Component
symfony/twig-bridge          v4.2.2  = v4.2.2  Symfony Twig Bridge
symfony/twig-bundle          v4.2.2  = v4.2.2  Symfony TwigBundle
symfony/validator            v4.2.2  = v4.2.2  Symfony Validator Component
symfony/var-dumper           v4.2.2  = v4.2.2  Symfony mechanism for exploring and dumping PHP variables
symfony/var-exporter         v4.2.2  = v4.2.2  A blend of var_export() + serialize() to turn any serializable data ...
symfony/web-profiler-bundle  v4.2.2  = v4.2.2  Symfony WebProfilerBundle
symfony/web-server-bundle    v4.2.2  = v4.2.2  Symfony WebServerBundle
symfony/yaml                 v4.2.2  = v4.2.2  Symfony Yaml Component
```

#### PHP version

```
PHP 7.2.0 (cli) (built: Nov 28 2017 23:48:49) ( ZTS MSVC15 (Visual C++ 2017) x64 )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2017 Zend Technologies
```

## Subject

The essence of the problem: you can not make a provider for the file (sonata.media.provider.file)

## Steps to reproduce
```
sonata_media:
    db_driver: doctrine_orm
    default_context: default
    contexts:
    project:
            providers:
                - sonata.media.provider.file

            formats: ~
```

## Actual results
Argument 3 passed to Sonata\MediaBundle\Provider\Metadata::__construct() must be of the type string or null, boolean given, called in C:\OSPanel\domains\render\vendor\sonata-project\media-bundle\src\Provider\FileProvider.php on line 70 (http://prntscr.com/ma3sne)

## Stack Trace

```
TypeError:
Argument 3 passed to Sonata\MediaBundle\Provider\Metadata::__construct() must be of the type string or null, boolean given, called in C:\OSPanel\domains\render\vendor\sonata-project\media-bundle\src\Provider\FileProvider.php on line 70

  at vendor/sonata-project/media-bundle/src/Provider/Metadata.php:46
  at Sonata\MediaBundle\Provider\Metadata->__construct('sonata.media.provider.file', 'sonata.media.provider.file.description', false, 'SonataMediaBundle', array('class' => 'fa fa-file-text-o'))
     (vendor/sonata-project/media-bundle/src/Provider/FileProvider.php:70)
  at Sonata\MediaBundle\Provider\FileProvider->getProviderMetadata()
     (vendor/twig/twig/lib/Twig/Extension/Core.php:1626)
  at twig_get_attribute(object(Twig_Environment), object(Twig_Source), object(FileProvider), 'providerMetadata', array())
     (var/cache/dev/twig/3b/3b1eac312c8cd73643990ef0e89555d26d6e2f832f388819a40a9b4e9b1c69a3.php:99)
  at __TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0->block_content(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables), '_parent' => array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), '_seq' => array(object(FileProvider)), '_iterated' => false, 'provider' => object(FileProvider), 'code' => 0), array('content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content'), 'preview' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_preview'), 'form' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_form'), 'list' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_list'), 'show' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_show'), 'actions' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_actions'), 'tab_menu' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_tab_menu'), 'title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title')))
     (vendor/twig/twig/lib/Twig/Template.php:189)
  at Twig_Template->displayBlock('content', array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content'), 'preview' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_preview'), 'form' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_form'), 'list' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_list'), 'show' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_show'), 'actions' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_actions'), 'tab_menu' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_tab_menu'), 'title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title')))
     (var/cache/dev/twig/e1/e13f574144d1be298e2f34a5910db0bef058a33e8e9e28d58599c5b9dc6e0218.php:37)
  at __TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832->doDisplay(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content'), 'preview' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_preview'), 'form' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_form'), 'list' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_list'), 'show' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_show'), 'actions' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_actions'), 'tab_menu' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_tab_menu'), 'title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title')))
     (vendor/twig/twig/lib/Twig/Template.php:386)
  at Twig_Template->displayWithErrorHandling(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content'), 'preview' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_preview'), 'form' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_form'), 'list' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_list'), 'show' => array(object(__TwigTemplate_9b4059d297598cd455138212d550ca60debabc08337c773c6d701f726129f832), 'block_show'), 'actions' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_actions'), 'tab_menu' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_tab_menu'), 'title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title')))
     (vendor/twig/twig/lib/Twig/Template.php:363)
  at Twig_Template->display(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('actions' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_actions'), 'tab_menu' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_tab_menu'), 'content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content'), 'title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title')))
     (var/cache/dev/twig/3e/3ed3ca5278f5f342b99feb2242cdf374412617df5934d1bb26a62d7ff93b64b1.php:35)
  at __TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc->doDisplay(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('actions' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_actions'), 'tab_menu' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_tab_menu'), 'content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content'), 'title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title')))
     (vendor/twig/twig/lib/Twig/Template.php:386)
  at Twig_Template->displayWithErrorHandling(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('actions' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_actions'), 'tab_menu' => array(object(__TwigTemplate_69667aaff61a2f041bdca03af731231e9cb5e833fa38f70fa2e0a6ff60dac7bc), 'block_tab_menu'), 'content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content'), 'title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title')))
     (vendor/twig/twig/lib/Twig/Template.php:363)
  at Twig_Template->display(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title'), 'content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content')))
     (var/cache/dev/twig/3b/3b1eac312c8cd73643990ef0e89555d26d6e2f832f388819a40a9b4e9b1c69a3.php:35)
  at __TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0->doDisplay(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title'), 'content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content')))
     (vendor/twig/twig/lib/Twig/Template.php:386)
  at Twig_Template->displayWithErrorHandling(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool), 'wrap_fields_with_addons' => true, 'app' => object(AppVariable), 'sonata_block' => object(GlobalVariables), 'sonata_admin' => object(GlobalVariables), 'sonata_media' => object(GlobalVariables)), array('title' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_title'), 'content' => array(object(__TwigTemplate_01c28ac26af496fa47ab522613d26ae1c1a75566083e866369996829a06257d0), 'block_content')))
     (vendor/twig/twig/lib/Twig/Template.php:363)
  at Twig_Template->display(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool)))
     (vendor/twig/twig/lib/Twig/Template.php:371)
  at Twig_Template->render(array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool)))
     (vendor/symfony/twig-bridge/TwigEngine.php:49)
  at Symfony\Bridge\Twig\TwigEngine->render('@SonataMedia/MediaAdmin/select_provider.html.twig', array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool)))
     (vendor/symfony/twig-bundle/TwigEngine.php:45)
  at Symfony\Bundle\TwigBundle\TwigEngine->render('@SonataMedia/MediaAdmin/select_provider.html.twig', array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool)))
     (vendor/symfony/templating/DelegatingEngine.php:41)
  at Symfony\Component\Templating\DelegatingEngine->render('@SonataMedia/MediaAdmin/select_provider.html.twig', array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool)))
     (vendor/symfony/framework-bundle/Controller/ControllerTrait.php:227)
  at Sonata\AdminBundle\Controller\CRUDController->originalRender('@SonataMedia/MediaAdmin/select_provider.html.twig', array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool)), null)
     (vendor/sonata-project/admin-bundle/src/Controller/CRUDController.php:142)
  at Sonata\AdminBundle\Controller\CRUDController->renderWithExtraParams('@SonataMedia/MediaAdmin/select_provider.html.twig', array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false), 'admin' => object(MediaAdmin), 'base_template' => '@SonataAdmin/ajax_layout.html.twig', 'admin_pool' => object(Pool)), null)
     (vendor/sonata-project/media-bundle/src/Controller/MediaAdminController.php:57)
  at Sonata\MediaBundle\Controller\MediaAdminController->render('@SonataMedia/MediaAdmin/select_provider.html.twig', array('providers' => array(object(FileProvider)), 'action' => 'create', 'media_pool' => object(Pool), 'persistent_parameters' => array('context' => 'project', 'category' => null, 'hide_context' => false)))
     (vendor/sonata-project/media-bundle/src/Controller/MediaAdminController.php:42)
  at Sonata\MediaBundle\Controller\MediaAdminController->createAction(object(Request))
     (vendor/symfony/http-kernel/HttpKernel.php:150)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:67)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:198)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:25)
```

**Comment by kunicmarko20** [comment]
This should be fixed with https://github.com/sonata-project/SonataMediaBundle/pull/1530, can you require 3.x-dev and see if it works for you and we will do a release soon?

**Comment by byhaskell** [comment]
Yes, thank you. I thought it was already poured into the release and did not work. Close the question

**Comment by byhaskell** [comment]
> This should be fixed with #1530, can you require 3.x-dev and see if it works for you and we will do a release soon?

@kunicmarko20 , Sorry, but the error has disappeared, but it is not possible to download the zip extension file.

**Comment by kunicmarko20** [comment]
sounds like a different issue, please open a new one.
